Watchman.DomainModel/Messages/Queries/Handlers/GetMessagesQueryHandler.cs

```diff
@@ -1,30 +1,35 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Text;
+﻿using System.Linq;
 using Watchman.Cqrs;
-using Watchman.DomainModel.Commons.Queries;
 using Watchman.DomainModel.Commons.Queries.Handlers;
 using Watchman.Integrations.MongoDB;
 
 namespace Watchman.DomainModel.Messages.Queries.Handlers
 {
     public class GetMessagesQueryHandler : PaginationQueryHandler, IQueryHandler<GetMessagesQuery, GetMessagesQueryResult>
     {
-        private readonly ISessionFactory sessionFactory;
+        private readonly ISessionFactory _sessionFactory;
 
         public GetMessagesQueryHandler(ISessionFactory sessionFactory)
         {
-            this.sessionFactory = sessionFactory;
+            this._sessionFactory = sessionFactory;
         }
 
         public GetMessagesQueryResult Handle(GetMessagesQuery query)
         {
-            using (var session = sessionFactory.Create())
+            using var session = _sessionFactory.Create();
+            var messages = session.Get<Message>();
+            if (query.ServerId == 0)
             {
-                var messages = session.Get<Message>();
-                var paginated = this.Paginate(query, messages);
-                return new GetMessagesQueryResult(paginated);
+                messages = TakeOnlyFromOneServer(query.ServerId, messages);
```

Comment:
jak to ma niby działać? 

ma filtrować tylko jak ktoś wybierze ID 0 
i ma nie znaleźć nic, bo żaden nie ma ID 0 
a jak ktoś ustawi normalne ID to ma zwrócić z wszystkich serwerów 

ServerID powinno być zawsze != 0 
jeśli jest == 0 powinien być wyjątek że nie może być zero 
lub powinien zwracać wtedy wszystkie, ale chyba lepszą opcją będzie wyjątek

Reply:
Napisałem == zamiast != 🤦‍♂️
Kiedy ServerId jest równy 0, zwracane są wszystkie wiadomości